flashcards/backend/decks/factories.py

```diff
@@ -46,15 +46,15 @@ class Meta:
         model = Deck
 
     image = ImageField(color="blue")
-    name = Faker("language_name")
-    category = SubFactory(CategoryFactory)
-    difficulty_level = SubFactory(DifficultyLevelFactory)
+    name: Faker = Faker("language_name")
+    category: SubFactory = SubFactory(CategoryFactory)
+    difficulty_level: SubFactory = SubFactory(DifficultyLevelFactory)
     is_public = True
-    price = Faker("pydecimal", right_digits=2, min_value=1, max_value=99)
-    author = SubFactory(UserFactory)
+    price: Faker = Faker("pydecimal", right_digits=2, min_value=1, max_value=99)
+    author: SubFactory = SubFactory(UserFactory)
     popularity = 0
     is_active = True
-    description = Faker("sentence", nb_words=8)
+    description: Faker = Faker("sentence", nb_words=8)
```

Comment:
So I'm still figuring out this whole static typing thing, and I know that this typing is enough for mypy to pass, but I was thinking if we could do sth like this, `name: Faker[str, Any] = Faker("language_name")`? Let me know what do you think.

Reply:
Ok, added additional annotation for input arguments for Faker, LazyAttribute and SubFactory

Reply:
When I'm adding argument type annotations and try to run custom commands for creating the test data I get the error "type 'Faker' is not subscriptable," . The explanation that I found is that 'Faker' class does not support subscripting or type parameterization directly. So, I left everything as it is.